### PR TITLE
fix: documentHost prefix added to reg. comment

### PR DESCRIPTION
**Pull request by ozankc**

### CODE REVIEW

File: packages/applications-service-api/src/controllers/representations.js [hunk 1]
```diff
@@ -72,10 +72,26 @@ module.exports = {
...
-			representation.dataValues.attachments = Object.values(attachments);
+			let attachments = await getDocumentsByDataId(dataIDs);
 
+			if (
+				attachments &&
+				attachments['0'] &&
+				Object.keys(attachments['0']).length === 0 &&
+				Object.getPrototypeOf(attachments['0']) === Object.prototype
+			) {
+				console.log('has no attachment');
```
Comment (Philip-Philippou1) [inline packages/applications-service-api/src/controllers/representations.js:86]: Can we use the logger here?

Comment (ozankc) [reply packages/applications-service-api/src/controllers/representations.js]: logger added, thanx

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**